.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This is needed until VS Code updates their TS version to 3.7 (in November release).